File: app/src/main/java/com/jajinba/pixabaydemo/network/ApiService.java
package com.jajinba.pixabaydemo.network;


import com.jajinba.pixabaydemo.model.PixabayResponseObject;

import java.util.Map;

import io.reactivex.Observable;
import retrofit2.http.POST;
import retrofit2.http.QueryMap;

interface ApiService {

  @POST("api")
  Observable<PixabayResponseObject> searchImages(@QueryMap Map<String, String> params);

}

File: app/src/main/java/com/jajinba/pixabaydemo/network/ApiService.kt
package com.jajinba.pixabaydemo.network

import com.jajinba.pixabaydemo.model.PixabayResponseObject
import io.reactivex.Observable
import retrofit2.Response
import retrofit2.http.POST
import retrofit2.http.QueryMap

internal interface ApiService {
    @POST("api")
    open fun searchImages(@QueryMap params: MutableMap<String?, String?>?): Observable<Response<PixabayResponseObject?>?>?
}